qt-bugs@ issue: none
Trolltech task ID: none
applied: no
author: from Trolltech

This patch fixes a rendering issue in bn_IN locale


--- src/gui/text/qscriptengine.cpp	Mon Jan  8 14:08:29 CET 2007
+++ src/gui/text/qscriptengine.cpp	Mon Jan  8 14:08:29 CET 2007

@@ -3055,7 +3057,7 @@
                         unsigned char pos = position[i];
                         for (int j = i+1; j > to+1; j--) {
                             uc[j] = uc[j-2];
-                            position[j] = uc[j-2];
+                            position[j] = position[j-2];
                         }
                         uc[to] = ch;
                         uc[to+1] = ch2;

 
